tests/wpt/web-platform-tests/html/semantics/forms/the-input-element/range.html

```diff
@@ -34,7 +34,7 @@ <h1>Input Range</h1>
       <input type="range" id="stepdown_beyond_min" min=3 max=11 value=6 step=3 />
       <input type="range" id="illegal_min_and_max" min="ab" max="f" />
       <input type="range" id="illegal_value_and_step" min=0 max=5 value="ppp" step="xyz" />
-      <input type="range" id="should_skip_whitespace" value=" 123"/>
+      <input type="range" id="should_skip_whitespace" value=" 98"/>
```

Comment:
Why use `98` rather than `123`?

Reply:
Because 123 is outside the default min-max range (0-100)